README.md

```diff
@@ -1 +1,51 @@
-# to-do-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Project Name 
+To Do List 
+
+## Project Description
+In this project I utilized
+- webpack to bundle JavaScript.
+- ES6 syntax to build to do list.
+- ES6 modules to write modular JavaScript.
+
+## Built With
+- HTML
+- CSS
+- Javascript 
+- Webpack
+
+## Deployment
+[Live view]()
```

Comment:
- Kindly add a valid link to the live demo or just give a note that it is under construction.